spring-boot-admin-docs/src/main/asciidoc/server.adoc

```diff
@@ -52,7 +56,7 @@ In addition when the reverse proxy terminates the https connection, it may be ne
 
 | spring.boot.admin.probed-endpoints
 | For Spring Boot 1.x client applications SBA probes for the specified endpoints using an OPTIONS request.
-  If the path differs from the id you can specify this as id:path (e.g. health:ping)..
+If the path differs from the id you can specify this as id:path (e.g. health:ping)..
```

Comment:
This change is not related to the PR, right? Can you please revert to removing the spaces?

Reply:
fixed in https://github.com/codecentric/spring-boot-admin/pull/1672/commits/60f3c408107f5af069ddf02245be59fb32970f62